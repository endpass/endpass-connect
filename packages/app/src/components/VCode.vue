<template>
  <pre class="code">
    <slot />
  </pre>  
</template>

<style lang="postcss">
  .code {
    padding: 20px;
    background-color: #dadada;
    overflow: auto;
    whitespace: nowrap;
  }
</style>