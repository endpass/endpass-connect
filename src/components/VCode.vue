<template>
  <pre class="code">
    <slot />
  </pre>
</template>

<script>
export default {
  name: 'VCode',
};
</script>

<style lang="postcss">
.code {
  padding: 20px;
  background-color: #dadada;
  overflow: auto;
  whitespace: nowrap;
}
</style>
