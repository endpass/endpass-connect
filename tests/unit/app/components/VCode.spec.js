import { shallowMount } from '@vue/test-utils';
import VCode from '@@/app/src/components/VCode.vue';

describe('VCode', () => {
  describe('render', () => {
    let wrapper;

    beforeEach(() => {
      wrapper = shallowMount(VCode);
    });

    it('should correctly render VCode component', () => {
      expect(wrapper.name()).toBe('VCode');
      expect(wrapper.html()).toMatchSnapshot();
    });
  });
});
