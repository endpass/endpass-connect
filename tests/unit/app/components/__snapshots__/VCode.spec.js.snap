// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VCode render should correctly render VCode component 1`] = `<pre class="code">  </pre>`;
